### ISSUE TITLE
Jak sprawdzić przyczynę odrzucenia oferty (stan INACTIVE) znając tylko numer oferty?

### ISSUE THREAD

**Issue by tomylee0** [allegro/allegro-api#9100]
Pytanie dotyczy #8832.

Czy jeśli nie zapiszę sobie "location" z odpowiedzi w chwili wysyłania POST /sale/product-offers to czy jest jakiś inny sposób żeby na podstawie przyznanego mi już numeru oferty i tylko numeru oferty sprawdzić jakiś czas później z jakiego powodu oferta pozostała w stanie INACTIVE? Chodzi mi o to, żebym nie musiał posiadać końcówki z url'a np.:
location: https://api.allegro.pl/sale/product-offers/<numer oferty>/operations/aa716f42...dc70e

Pozdrawiam

**Comment by Lukasz-Zurek** [comment]
Nie, nie ma innego sposobu, by na podstawie samego numeru oferty sprawdzić szczegóły dotyczące pozostania oferty w statusie INACTIVE.
